### PR TITLE
Incorporate payouts and move out-of-scope text

### PR DESCRIPTION
**Pull request by gene1wood**
This adds in the payouts details, examples of each severity and severity descriptions. It also moves the listed out-of-scope details from the policy page to the bounty page.

### CODE REVIEW

File: mozilla_core_services/policy.md [hunk 1]
```diff
@@ -2,23 +2,25 @@ The Mozilla Bug Bounty Program is designed to encourage security research into M
 
 # Program Scope
 
-Please check the list of sites under the Scope section, we would like testing to focus on those sites. Other sites and services are considered out of scope of the program unless you think the bug is critical. Please check our [website](https://www.mozilla.org/en-US/security/web-bug-bounty/) for examples of what we consider critical issues.
+Please check the list of sites under the Scope section, we would like testing to focus on those sites. Other sites and services are considered out of scope of the program unless you think the bug is critical (see above for examples of critical issues)
```
Comment (fkiriakos07) [inline mozilla_core_services/policy.md]:
```suggestion
Please check the list of sites under the Scope section, we would like testing to focus on those sites. Other sites and services are considered out of scope of the program unless the bug is critical (see above for examples of critical issues)
```

File: mozilla_critical_services/policy.md [hunk 2]
```diff
@@ -2,23 +2,25 @@ The Mozilla Bug Bounty Program is designed to encourage security research into M
 
 # Program Scope
 
-Please check the list of sites under the Scope section, we would like testing to focus on those sites. Other sites and services are considered out of scope of the program unless you think the bug is critical. Please check our [website](https://www.mozilla.org/en-US/security/web-bug-bounty/) for examples of what we consider critical issues.
+Please check the list of sites under the Scope section, we would like testing to focus on those sites. Other sites and services are considered out of scope of the program unless you think the bug is critical (see above for examples of critical issues)
```
Comment (fkiriakos07) [inline mozilla_critical_services/policy.md]:
```suggestion
Please check the list of sites under the Scope section, we would like testing to focus on those sites. Other sites and services are considered out of scope of the program unless the bug is critical (see above for examples of critical issues)
```

**Review by fkiriakos07 — COMMENTED** [review]
added a couple of suggestions based on our discussion yesterday. Otherwise, it looks good to me. Thanks Gene.

**Comment by gene1wood** [comment]
I've deployed these changes

**Pull request merged**